src/platform/wifi/nm-wifi-utils-nl80211.c

```diff
@@ -958,46 +947,35 @@ nm_wifi_utils_nl80211_new (int ifindex, struct nl_sock *genl)
 
 	msg = nl80211_alloc_msg (nl80211, NL80211_CMD_GET_WIPHY, 0);
 
+	device_info.nl80211 = nl80211;
 	if (nl80211_send_and_recv (nl80211, msg, nl80211_wiphy_info_handler,
 	                           &device_info) < 0) {
-		_LOGD (LOGD_PLATFORM | LOGD_WIFI,
-		       "(%s): NL80211_CMD_GET_WIPHY request failed",
-		       ifname);
+		_LOGD (LOGD_PLATFORM | LOGD_WIFI, "NL80211_CMD_GET_WIPHY request failed");
```

Comment:
since all callers use the same logging domain, `_NMLOG()` should not have this parameter explicitly. See for examle `src/devices/nm-acd-manager.c`. It should be something like:

```C
#define _NMLOG_DOMAIN             (LOGD_PLATFORM | LOGD_WIFI)  
#define _NMLOG_PREFIX_NAME        "wifi-nl80211"
#define _NMLOG(level, ...) \ 
```

Reply:
Note that I didn't change anything about the logging domains.

Reply:
You refactor `_NMLOG()`. At that point, it would be good to also get rid of the domain argument and move it inside the macro. Like done at most other places (notable exception, NMDevice, which actually wants to log with different domains).

Reply:
Okay...